src/constants/triggers.constants.ts

```diff
@@ -15,12 +15,12 @@ export let extraTriggerTypes = [
 i18n.on("initialized", () => {
 	infoCronExpressionsLinks = [
 		{
-			text: i18n.t("triggers.form.info.aboutCron", { ns: "tabs" }),
-			url: "https://docs.oracle.com/cd/E12058_01/doc/doc.1014/e12030/cron_expressions.htm",
+			text: i18n.t("triggers.form.info.firstLink", { ns: "tabs" }),
+			url: "https://crontab.guru/",
 		},
 		{
-			text: i18n.t("triggers.form.info.schedules", { ns: "tabs" }),
-			url: "https://docs.temporal.io/workflows#robfig-predefined-schedules-and-intervals",
+			text: i18n.t("triggers.form.info.secondLink", { ns: "tabs" }),
+			url: "https://github.com/autokitteh/kittehub/tree/main/samples/scheduler#api-documentation",
```

Comment:
Instead of one link to the docs, place these three links:
[Cron expression format](https://pkg.go.dev/github.com/robfig/cron/v3#hdr-CRON_Expression_Format) ("* * * * *")

[Cron extended expression format](https://pkg.go.dev/github.com/robfig/cron/v3#hdr-Alternative_Formats) (includes seconds and minutes)

[Predefined schedules and intervals](https://pkg.go.dev/github.com/robfig/cron#hdr-Predefined_schedules) ("@" format)

Reply:
done